saas/api/metrics.py

```diff
@@ -54,6 +55,9 @@ class BalancesAPIView(DateRangeContextMixin, ProviderMixin,
     Generate a table of revenue (rows) per months (columns) for a default
     balance sheet (Income, Backlog, Receivable).
 
+    Also supports other time periods to retrieve data for: hourly, daily,
```

Comment:
Define `PeriodSerializer` and add a decorator on the `get` method such as:
```
    @swagger_auto_schema(query_serializer=PeriodSerializer)
```
such that the [OpenAPI schema](https://swagger.io/specification/) can be automatically generated.

You can see a model of this in the definition and use of `ForceSerializer`.